Shared/Tests.Sync.Shared/PermissionsTests.cs

```diff
@@ -47,5 +50,35 @@ public void PermissionChange_ShouldNotBeInDefaultSchema()
         {
             Assert.That(RealmSchema.Default.Find(nameof(PermissionChange)), Is.Null);
         }
+
+        [Test, Explicit]
```

Comment:
Does this work (localhost login) for both iOS and Android simulators? What about device testing? If the test has constraints please add them to the `Explicit` attribute